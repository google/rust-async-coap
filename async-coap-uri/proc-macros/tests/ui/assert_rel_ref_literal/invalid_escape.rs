use proc_macro_hack::proc_macro_hack;

#[proc_macro_hack]
use async_coap_uri_macros::assert_rel_ref_literal;

fn call_macro() {
    assert_rel_ref_literal!("https://www.example.com/a/%:A/c");
}

fn main() {}
