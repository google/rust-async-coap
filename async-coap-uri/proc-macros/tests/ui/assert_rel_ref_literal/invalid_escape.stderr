error: the 2 char after '%' must be valid hex character
 --> $DIR/invalid_escape.rs:7:29
  |
7 |     assert_rel_ref_literal!("https://www.example.com/a/%:A/c");
  |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
